lib/rubocop/cop/naming/method_name.rb

```diff
@@ -28,12 +21,41 @@ module Naming
       #
       #   # good
       #   def fooBar; end
+      #
+      # The cop can also be configured to `AllowedPatterns` to allow certain patterns:
+      #   Naming/MethodName:
+      #     AllowedPatterns:
+      #       - '\AonSelectionBulkChange\z'
+      #       - '\AonSelectionCleared\z'
+      #
+      # Similarly you can also forbid specific method names or patterns
+      # using `ForbiddenIdentifiers` or `ForbiddenPatterns`.
+      #
+      #   Naming/MethodName:
+      #     ForbiddenIdentifiers:
+      #       - 'def'
+      #       - 'super'
+      #     ForbiddenPatterns:
+      #       - '_v1\z'
+      #       - '_gen1\z'
```

Comment:
```suggestion
      # The cop can be configured with `AllowedPatterns` to allow certain regexp patterns:
      #
      # [source,yaml]
      # ----
      # Naming/MethodName:
      #   AllowedPatterns:
      #     - '\AonSelectionBulkChange\z'
      #     - '\AonSelectionCleared\z'
      # ----
      #
      # As well, you can also forbid specific method names or regexp patterns
      # using `ForbiddenIdentifiers` or `ForbiddenPatterns`:
      #
      # [source,yaml]
      # ---- 
      # Naming/MethodName:
      #   ForbiddenIdentifiers:
      #     - 'def'
      #     - 'super'
      #   ForbiddenPatterns:
      #     - '_v1\z'
      #     - '_gen1\z'
      # ----
```

Also, the comments should be above all `@example` blocks.